js/packages/fullstack-utils/src/Utils/parser.ts

```diff
@@ -0,0 +1,71 @@
+import type {
+  LoaderInterface,
+  PlayerAuthJwt,
+  AuthRequestInterface,
+} from "@nirvai-web/types";
+import { isOld } from "./datetime";
+
+import { defaultLoaderParams } from "../Loaders";
+
+export const fromUrl = (x = "") => x.replace(/-/g, " ").trim();
+export const toUrl = (x = "") => x.replace(/\s/g, "-").trim();
+export const toPretty = (x = "") => x.replace(/[^A-Za-z0-9]/g, " ").trim();
+
+/**
+ * @see LoaderInterface
+ */
+const parsableKeys = [
```

Comment:
i think we used this in nirv to massage UI fields in backend db fields

we should either delete this entire section or make it an example